src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -740,15 +748,6 @@ impl A320HydraulicLogic {
         }
     }
 
-    fn update_external_cond(&mut self, delta_time_update: &Duration) {
-        // Only evaluate ground conditions if on ground, if superman needs to operate cargo door in flight feel free to update
-        if self.weight_on_wheels {
```

Comment:
Updating these things while in the air is in fact important. For example, in the unlikely event that someone pushed back and takes off within the NWS pin inserted timeout, that condition would remain `true` until after aircraft landing.

Now the `DelayedFalseLogicGate` simply hides these time related difficulties. Let's keep updating it! 😀

Reply:
That would be a nasty usage of the plane :)   No problem but when performance becomes a problem that could still be something to consider

Reply:
It's very unlikely that these things have the highest performance impact. So no worries.